### PR TITLE
Removed unused clock from Profiler

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Fixes 1) in https://github.com/cogneco/ooc-kean/issues/716
@bettinaselig can you review ?

### CODE REVIEW

**Comment by bettinaselig** [comment]
Reviewed.

File: source/base/Profiler.ooc [hunk 1]
```diff
@@ -51,8 +48,6 @@ Profiler: class {
 			}
 		this _timer free()
 		this _timer = null
```
Comment (marcusnaslund) [inline source/base/Profiler.ooc]: Is there any reason to do this?

Comment (sebastianbaginski) [inline source/base/Profiler.ooc]: None that I'm aware of.

Comment (marcusnaslund) [inline source/base/Profiler.ooc]: In that case, since it's just a one-line removal, and this is a cleanup PR, please delete it here.

Comment (sebastianbaginski) [inline source/base/Profiler.ooc]: Done.

**Pull request merged**